tests/test_env.py

```diff
@@ -288,6 +288,36 @@ def test_process_env_install_from_index_not_found_env_var(
         )
 
 
+def test_process_env_install_no_index(modules_v2_dir) -> None:
```

Comment:
Why use the `modules_v2_dir` fixture here?

Reply:
I forgot to remove it, I first tried using this instead of creating the package myself. I removed it now, thanks